docs/content/guides/cell-functions/cell-editor.md

```diff
@@ -61,32 +210,32 @@ class CustomEditor extends TextEditor {
   }
 }
 
-const hotSettings = {
-  startRows: 5,
-  columns: [
-    {
-      editor: CustomEditor
-    }
-  ],
-  colHeaders: true,
-  colWidths: 200,
-  licenseKey: 'non-commercial-and-evaluation'
-};
-
 const ExampleComponent = () => {
   return (
-    <div>
-      <HotTable
-        id="hot"
-        settings={hotSettings}
-      />
-    </div>
+    <HotTable
+      id="hot"
+      startRows={5}
+      columns={[
+        {
+          editor: CustomEditor
+        }
+      ]}
+      colHeaders={true}
+      colWidths={200}
+      licenseKey="non-commercial-and-evaluation"
+    />
   );
 }
 
-ReactDOM.render(<ExampleComponent />, document.getElementById('example1'));
+ReactDOM.render(<ExampleComponent />, document.getElementById('example2'));
 ```
 :::
+
+::: tip
+All the sections below describe how to utilize the features available for the Handsontable class-based editors.
+This information is applicable in React when using the non-component editor approach.
+:::
```

Comment:
```suggestion
::: tip
All the sections below apply to [class-based editors](#class-based-editors), not to [component-based editors](#component-based-editors).
:::
```

If this is true, instead of the note, all those sections should be nested inside the **Declaring an editor as a class** (AKA **Class-based editors**) section.

Reply:
I wanted to do that, but decreasing the levels of all of those headers messed with the TOC ([I even asked about it on Slack](https://handsoncode.slack.com/archives/C034WPQUG9W/p1659705625483469)).

Should I move it? WDYT?

Reply:
The right-hand TOC is meant to show 2 levels of headings by design. I'd move it.

Reply:
@kirszenbaum Done in https://github.com/handsontable/handsontable/pull/9752/commits/978e3eff7168b7b2d4b07beded1adb401a5fb32d. 

Additionally, I added `font-size` for the lower-level headers (namely, `H4` and `H5`), so the `H5` headers wouldn't be smaller than the text in the paragraphs below them, which looked pretty ridiculous.
As far as I can tell, there were no `H5`s in any of the articles yet, and the size of `H4` was modified marginally (to give some gradation to the header sizes), so I don't think it should break anything on the other pages.

Reply:
@jansiegel - looks great, thanks!